### PR TITLE
Airflow 2.8.0 blog post

### PR DESCRIPTION
**Pull request by ephraimbuddy**

### CODE REVIEW

**Review by potiuk — APPROVED** [review]
LGTM. But I would add two things:

1)  separate section for  a bunch of UI improvements that were added. Each one separately is small but put together things like "multi-select " on state (some of htem have  videos and screenshots that can be used in the blog post)


* https://github.com/apache/airflow/pull/34567
* https://github.com/apache/airflow/pull/35284
* https://github.com/apache/airflow/pull/35668
* https://github.com/apache/airflow/pull/35403
* https://github.com/apache/airflow/pull/34486 - this is also a good example to show - recently there was a discussion with angry users "bring old view" back and one of the issues raised was status visibility. it could be great to use this as an opportunity to show "we are listening and improving our UI" and "If you have any more requests how it can be improved rais an issue or PR


2) I'd also mention disabling removal of HTML form as part of our continuous quest to make airflow more secure. It's important to say that we are doing it because we care for security of our users and "secure by default" is one of the things we follow strongly.

File: landing-pages/site/content/en/blog/airflow-2.8.0/index.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
+---
+title: "Apache Airflow 2.8.0 is here"
+linkTitle: "Apache Airflow 2.8.0 is here"
+author: "Ephraim Anierobi"
+github: "ephraimbuddy"
+linkedin: "ephraimanierobi"
+description: "Introducing Apache Airflow 2.8.0: Enhanced with New Features and Significant Improvements"
+tags: [Release]
+date: "2023-12-14"
```
Comment (phanikumv) [inline landing-pages/site/content/en/blog/airflow-2.8.0/index.md:9]:
```suggestion
date: "2023-12-15"
```

File: landing-pages/site/content/en/blog/airflow-2.8.0/index.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+information within a single log view.
+
+Presently, suppose a task is terminated by the scheduler before initiation, times out due to prolonged queuing, or transitions into a zombie state. In that case, it is not recorded in the task log. With this enhancement, in such situations,
+it becomes feasible to dispatch an error message to the task log for convenient visibility on the UI.
+
+This feature can be toggled, for more information [Look for “enable_task_context_logger” in the logging configuration documentation](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#logging)
+
+## Listener hooks for Datasets
+This feature enables users to subscribe to Dataset creation and update events using listener hooks.
+It’s particularly useful to trigger external processes based on a Dataset being created or updated.
```
Comment (phanikumv) [inline landing-pages/site/content/en/blog/airflow-2.8.0/index.md:73]: this should be marked experimental?

Comment (ephraimbuddy) [reply landing-pages/site/content/en/blog/airflow-2.8.0/index.md:74]: Yeah, have added it below

**Review by potiuk — APPROVED** [review]
Fantastic!

File: landing-pages/site/content/en/blog/airflow-2.8.0/index.md [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+This feature seamlessly integrates task-related messages from various Airflow components, including the Scheduler and
+Executors, into the task logs. This integration allows users to easily track error messages and other relevant
+information within a single log view.
+
+Presently, suppose a task is terminated by the scheduler before initiation, times out due to prolonged queuing, or transitions into a zombie state. In that case, it is not recorded in the task log. With this enhancement, in such situations,
+it becomes feasible to dispatch an error message to the task log for convenient visibility on the UI.
+
+This feature can be toggled, for more information [Look for “enable_task_context_logger” in the logging configuration documentation](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html#logging)
+
+## Listener hooks for Datasets
```
Comment (phanikumv) [inline landing-pages/site/content/en/blog/airflow-2.8.0/index.md:72]: * Please note that listeners are still experimental and subject to change. *

File: landing-pages/site/content/en/blog/airflow-2.8.0/index.md [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+This feature enables users to subscribe to Dataset creation and update events using listener hooks.
+It’s particularly useful to trigger external processes based on a Dataset being created or updated.
+
+Please note that listeners are still experimental and subject to change.
```
Comment (phanikumv) [inline landing-pages/site/content/en/blog/airflow-2.8.0/index.md:76]:
```suggestion
```

File: landing-pages/site/content/en/blog/airflow-2.8.0/index.md (hunk 3 above)
Comment (phanikumv) [reply landing-pages/site/content/en/blog/airflow-2.8.0/index.md:72]: Lets add this call out just below the header

File: landing-pages/site/content/en/blog/airflow-2.8.0/index.md (hunk 4 above)
Comment (phanikumv) [reply landing-pages/site/content/en/blog/airflow-2.8.0/index.md:76]: Lets add this call out just below the header

**Review by phanikumv — APPROVED** [review]

**Pull request merged**